src/components/Search/Search.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import Overlay from '../UI/Overlay';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMagnifyingGlass } from '@fortawesome/free-solid-svg-icons';
+
+const Search = ({ closeModal }) => {
+  return (
+    <Overlay onClick={closeModal}>
+      <div className="search">
+        <div className="search__controls">
+          <input
+            className="search__controls__input"
+            type="text"
+            placeholder="Enter your search keyword..."
+          />
+          <button className="search__controls__button">
+            <FontAwesomeIcon icon={faMagnifyingGlass} />
+          </button>
```

Comment:
It does nothing, so it shouldn't be button

Reply:
I add function which will start search in the future